src/CommonGLPI.php

```diff
@@ -606,7 +606,7 @@ public static function getMenuName()
      * @param CommonGLPI $item         Item on which the tab need to be displayed
      * @param integer    $withtemplate is a template object ? (default 0)
      *
-     *  @return string tab name
+     *  @return string|array tab name
```

Comment:
I'm not sure this method should return an array

Reply:
It permit to add multiple tabs for a given itemtype, see https://github.com/glpi-project/glpi/blob/51387d7f93c2f7ed1615681b9e45c7ecfc400dc5/src/AuthLDAP.php#L3938-L3955

Reply:
By the way, it has already been fixed in the main branch : https://github.com/glpi-project/glpi/commit/f199f846c0eaa0e80d9b8c2dbd0b9eb723071537

```suggestion
     *  @return string|string[] tab name
```

Reply:
Avec ceci cela remonte plusieurs lint : 
**should return string but returns array<int, string>**